File: JS-Advanced/JS-Advanced/L01SyntaxFunctions/Lab/dayOfWeek.js
function day(day){
    let dayNumber;

    if(day == 'Monday'){
        dayNumber = 1;
    } else if(day == 'Tuesday'){
        dayNumber = 2;
    } else if(day == 'Wednesday'){
        dayNumber = 3;
    } else if(day == 'Thursday'){
        dayNumber = 4;
    } else if(day == 'Friday'){
        dayNumber = 5;
    } else if(day == 'Saturday'){
        dayNumber = 6;
    } else if(day == 'Sunday'){
        dayNumber = 7;
    } else{
        dayNumber = 'error';
    }

    console.log(dayNumber);
}

day('Monday');
day('Sunday');
day('asdad');

File: Задачи/Arrays - Lab/Arrays - Lab/DayOfWeek.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Arrays___Lab
{
    class DayOfWeek
    {
        static void Main(string[] args)
        {
            int dayNumber = int.Parse(Console.ReadLine());

            string[] daysOfWeek = new string[] { "Monday","Tuesday","Wednesday","Thursday","Friday","Saturday","Sunday" };

            if (dayNumber > 0 && dayNumber <= 7)
            {
                Console.WriteLine(daysOfWeek[dayNumber-1]);
            }
            else
            {
                Console.WriteLine("Invalid day!");
            }
        }
    }
}
